src/wiktextract/clean.py

```diff
@@ -1423,9 +1439,12 @@ def repl_1_syntaxhighlight(m):
     title = re.sub(r"\[//[^]\s]+\s+edit\s*\]", "", title)
     # Replace links by their text
 
-    category_ns_data = wxr.wtp.NAMESPACE_DATA.get("Category", {})
-    category_ns_names = {category_ns_data.get("name")} | set(
-        category_ns_data.get("aliases")
+    category_ns_data: Optional[NamespaceDataEntry]
+    # XXX "Category" -> config variable for portability
+    category_ns_data = wxr.wtp.NAMESPACE_DATA.get("Category", None)
```

Comment:
`dict.get()` returns `None` by default if a default value is not passed.

Reply:
I guess that is true, but it feels naughty.